spec/factories/parks.rb

```diff
@@ -2,7 +2,7 @@
   factory :park do
     sequence(:title) { |n| "park no. #{n}" }
     sequence(:location) { |n| "Location#{n}" }
-    rating 10
+    rating { rand(11) }
```

Comment:
Having random results in your test suite is not advised. This can create difficult to reproduce problems.